.gitignore

```diff
@@ -101,3 +101,5 @@ version.ml
 version.texi
 .cargo
 !src/vendor/**
+**/target/rls/debug/
```

Comment:
I think with rls in the same workspace as everything else wnow (a relatively recent change) this may no longer be necessary?